api/map-feature/map-feature.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="color-scheme" content="dark light">
+    <title>&lt;map-feature&gt; API</title>
+    <script type="module" src="../../dist/mapml-viewer.js"></script>
+    <style>
+      body {
+        font-family: 'system-ui', sans-serif;
+        max-width: 50rem;
+        margin: 0 auto;
+        padding: 0 1rem;
+        overflow-wrap: break-word;
+      }
+      mapml-viewer {
+        width: 100%;
+        min-height: 50vh;
+      }
+      textarea {
+          width: 100%;
+          min-height: 35vh;
+          margin-top: 10px;
+      }
+      @media (prefers-color-scheme: dark) {
+        .user-preference-message {
+          display: none;
+        }
+      }
+    </style>
+    <script>
+//      function addEmptyLayer() {
+//        let layer = document.querySelector('#empty-layer').content.firstElementChild.cloneNode(true);
+//        layer.label = 'Two Hat Man';
+//        layer.checked = 'checked';
+//        let map = document.querySelector('mapml-viewer');
+//        map.appendChild(layer);
+//      }
+//      addEmptyLayer();
+      function addMan() {
+        let map = document.querySelector('mapml-viewer');
+        let man = document.querySelector('#hatman').content.firstElementChild.cloneNode(true);
+        let layer = document.querySelector("#target-layer");
+        layer.appendChild(man);
+        layer.label = "Two Hat Man";
+        // this step should not be necessary
+        layer.remove();
```

Comment:
Nice catch, I think this may be due to the fact that the layer is empty at the start. Using the pasteFeature for the layer also does not work on an empty layer.

ex. 

```js
$0.pasteFeature('<map-feature><map-featurecaption>test</map-featurecaption><map-geometry><map-point><map-coordinates>-79.49047813600689 43.66921938354113</map-coordinates></map-point></map-geometry><map-properties><table><thead><tr><th role="columnheader" scope="col">Property name</th><th role="columnheader" scope="col">Property value</th></tr></thead><tbody></tbody></table></map-properties></map-feature>')
```